Update/help links (#9741)
getsentry/sentry-docs: docs/product/accounts/quotas/manage-event-stream-guide.mdx

Once your event volume is approaching or has exceeded the quota, teammates with the "Owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. You can then choose to [increase](#increasing-quotas) or [decrease](#decreasing-quotas) your quota.

> If this is your first time exceeding errors quota and you're on a paid plan, however, you'll be entered into a one-time grace period. Learn more about the grace period in this [Help article](https://sentry.zendesk.com/hc/en-us/articles/23983502355099-What-happens-when-I-run-out-of-event-capacity-and-a-grace-period-is-triggered).

Reviewer: ```suggestion
If this is your first time exceeding your errors quota and you're on a paid plan, you'll be entered into a one-time grace period. Learn more by reading this [help article](https://sentry.zendesk.com/hc/en-us/articles/23983502355099-What-happens-when-I-run-out-of-event-capacity-and-a-grace-period-is-triggered).
```
